docs/dev/dev_environment.md

```diff
@@ -37,6 +37,7 @@ Once you have Poetry and Docker installed you can run the following commands (in
 ```shell
 poetry shell
 poetry install
+cp development/dev.example.env development/dev.env
```

Comment:
Our standard is to move everything into creds.env, I wouldn't want to change that for this use case.

Reply:
Is it reasonable then to remove the reference in the documentation to `dev.env` if everything is supposed to be in `creds.env`?

https://github.com/nautobot/nautobot-plugin-golden-config/blob/d0b6fc343230fef69949d5b935398480411c657e/docs/dev/dev_environment.md?plain=1#L69

Reply:
We have a [development.env](https://github.com/nautobot/nautobot-plugin-golden-config/blob/develop/development/development.env) file that is used to store "non-secret" configuration settings for the development environment, creds.env should be used to store any passwords/actual secret information. The command reference above is indeed incorrect and should be `export $(cat development/development.env | xargs)`

Reply:
@jlixfeld any additional questions on this?

Reply:
Nope, but I'm not sure what the expectation is at this point.

Reply:
Remove everything except `cisco_xr` from this PR.